int32 id
uint32 max_orientations
float64 confidence

Orientation[] orientations
